src/start-sc.ts

```diff
@@ -38,19 +37,29 @@ function buildOptions(): string[] {
             required: optionMapping.required
         })
 
-        if (input === '') {
-            // user input nothing for this option
-        } else if (optionMapping.flag) {
+        if (input === '' && optionMapping.actionOption === 'logFile') {
+            params.push(`--logfile=${LOG_FILE}`)
+        }else if (optionMapping.flag) {
             // for boolean flag options like `--tunnel-pool`
             params.push(`--${optionMapping.scOption}`)
-        } else {
+        } else if(input !== '') {
             params.push(`--${optionMapping.scOption}=${input}`)
         }
     }
     return params
 }
 
 export async function startSc(): Promise<string> {
+    try {
+        tmp = mkdtempSync(join(tmpdir(), `sauce-connect-action`))
+        LOG_FILE = join(tmp, 'sauce-connect.log')
+        READY_FILE = join(tmp, 'sc.ready')
+        openSync(LOG_FILE, 'w');
```

Comment:
Will `existsSync` work on just the tmp folder? While this works to create the folder structure, the log message is misleading if the user doesn't use the default `LOG_FILE`.

Reply:
Maybe something like:
```
if !existsSync(TEMP_FOLDER) {
  mkdtempSync(TEMP_FOLDER)
}
```

Reply:
Or, change the log message to `Error writing to temp directory`

Reply:
If `--logfile` is added as a parameter, then the default log file is still used in error reporting below